README.md

```diff
@@ -84,7 +84,7 @@ and [Report Transaction](https://dev.maxmind.com/minfraud/report-transaction/) A
 ### minFraud API ###
 
 To use the minFraud API, create a new `\MaxMind\MinFraud` object. The constructor
-takes your MaxMind account ID, license key, and an optional options array as
+takes your MaxMind account ID, license key, and an optional [options](#Options) array as
```

Comment:
I think we could link to the API docs that have separate docs about these in the README - see [here](https://maxmind.github.io/minfraud-api-php/doc/v1.23.0/classes/MaxMind-MinFraud.html#method___construct).

For GeoIP2 APIs, we reference the other hosts [like so](https://github.com/maxmind/GeoIP2-php/blob/543732f574e55de80aaf3c934de8e4fd931b00e6/README.md?plain=1#L286-L293). Could we do the same here for the sandbox?